README.md

```diff
@@ -5,21 +5,97 @@ Right now this app only works on android. iOS support is planned in the future.
 
 ## Project setup
 
-_This project assumes that the machine running ths code has JDK, Android SDK and other basic dependencies install to run an Android project._
+The instructions provided below are for macOS. Detailed setup instructions for react-native projects along with dependencies for other operating systems can be found [here](https://reactnative.dev/docs/environment-setup?guide=native&platform=android)
+
+### Node and JDK setup
 
 This app is a react-native based app with java bridges to interface between android core API's and the react-native component.
-Basic setup instructions for react-native projects along with dependencies can be found [here](https://reactnative.dev/docs/environment-setup?guide=native&platform=android)
-Please follow this guide exactly.
+Install node and watchman. If you have already installed Node on your system, make sure it is Node 18 or newer. Watchman is a tool by Facebook for watching changes in the filesystem. It is highly recommended you install it for better performance.
```

Comment:
Install node and watchman. Make sure to use Node 18 or higher version. Watchman, a tool by Facebook, allows watching changes in the filesystem. It is highly recommended to install it for better performance.